pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun.go

```diff
@@ -211,35 +209,17 @@ func verifyTaskOCIBundle(
 		return apis.ErrGeneric(err.Error()).ViaKey(bundle)
 	}
 
-	digest, err := digest(ctx, bundle, kc)
-	if err != nil {
-		return apis.ErrGeneric(err.Error()).ViaKey(bundle)
-	}
-
-	if err := verifier.VerifySignature(bytes.NewReader(signature), bytes.NewReader([]byte(digest))); err != nil {
-		return apis.ErrGeneric(err.Error()).ViaKey(bundle)
-	}
-
-	return nil
-}
-
-func digest(ctx context.Context, imageReference string, keychain authn.Keychain) (string, error) {
 	timeoutCtx, cancel := context.WithTimeout(ctx, time.Second*60)
 	defer cancel()
 
-	imgRef, err := imgname.ParseReference(imageReference)
+	digest, err := Digest(ctx, bundle, remote.WithAuthFromKeychain(kc), remote.WithContext(timeoutCtx))
 	if err != nil {
-		return "", err
+		return apis.ErrGeneric(err.Error()).ViaKey(bundle)
```

Comment:
If this is different behavior, consider breaking this out into its own smaller PR.

Reply:
You mean the comparisons? I think this might be a github/git issue? It's making comparisons on wrong functions, so what I did is to move `digest()` from this file to `sign.go `